src/igl/vulkan/util/TextureFormat.cpp

```diff
@@ -6,6 +6,7 @@
  */
 
 #include <igl/vulkan/util/TextureFormat.h>
+#include <igl/vulkan/Common.h>
```

Comment:
The purpose of `util/TextureFormat.cpp` is to be used where Vulkan headers are not compiled in. That's why we need this macro definitions.

Reply:
done